src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,15 @@
-const fizzBuzz = number => {};
+const fizzBuzz = (number) => {
+    if (number % 5 === 0 && number % 3 === 0) {
```

Comment:
Nice work on this. Extra challenge - can you refactor it to use the lowest common multiple of 3 and 5 so this if statement only contains one === statement?